HTML/index.html

```diff
@@ -0,0 +1,794 @@
+              <div class="row justify-content-center cu-row">
+                <div class="col-md-4 cu-col">
+                  <div class="cu-inner-col">
+
+                  <i class="bi bi-geo-alt-fill cu-icon"></i>
+                  <p class="contact-head">ADDRESS</p>
+                  <p class="contact-add">Level 6, 23 pitt St, Sydney</p>
+                  </div>
+                </div>
+                <div class="col-md-4 cu-col">
+                  <div class="cu-inner-col">
+                    <i class="bi bi-phone cu-icon"></i>
+                    <p class="contact-head">PHONE</p>
+                    <p>+61 9 211 3741</p>
+                  </div>
+                
+                </div>
+                <div class="col-md-4 cu-col">
+                  <div class="cu-inner-col">
+
+                  <i class="bi bi-cursor cu-icon"></i>
+                  <p class="contact-head">EMAIL</p>
+                  <p class="text-dark">hey@halcyon.com.au</p>
+                </div>
+                </div>
+              </div>
+            </div>
+            <div class="row cu-btn-con justify-content-center">
+              <div class="col-lg-3">
+                <div class="row justify-content-center ">
+                  <a href="#" class="col-4 cu-icon-con"><i class="bi bi-globe cu-icons"></i></a>
+                  <a href="#" class="col-4 cu-icon-con"
+                    ><i class="bi bi-twitter cu-icons"></i
+                  ></a>
+                  <a href="#" class="col-4 cu-icon-con"
+                    ><i class="bi bi-envelope cu-icons"></i
+                  ></a>
+                </div>
+              </div>
+            </div>
+         
+        </div>
+        </div>
+
+    <!-- footer  -->
+    <div class="footer">
+    <div class="container">
+      <div class="row justify-content-center">
+        <div class="col-5 tandc">
+          <p class="f-terms">
+            <a href="" class="f-terms">Terms & Conditions</a> |
+            <a href="" class="f-terms">Legals</a>
+          </p>
+        </div>
+
+        <div class="col-5 f-right">
+          Designed & Developed by
+          <span class="f-terms">Peter Finian </span> exclusivly for<span
+            class="f-terms"
+          >
+            Codrops</span
+          >
+        </div>
+      </div>
+    </div>
+    </div>
+    <!-- js  -->
+    <script
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
+      integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
+      crossorigin="anonymous"
+    ></script>
+
+    <!-- slick js  -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.6.1/jquery.min.js" integrity="sha512-aVKKRRi/Q/YV+4mjoKBsE4x3H+BkegoM/em46NNlCqNTmUYADjBbeNefNxYV7giUp0VxICtqdrbqU7iVaeZNXA==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+    <script type="text/javascript" src="../slick/slick.min.js"></script>
+
+    <script>
```

Comment:
please removed js code from html file and move to separate js file

Reply:
included JS file